rosdep/python.yaml

```diff
@@ -701,6 +701,12 @@ python-falcon:
   fedora: [python-falcon]
   ubuntu: [python-falcon]
 python-fcn-pip:
+  debian:
+    pip:
+      packages: [fcn]
```

Comment:
Does the debian rule not have the same depends as the ubuntu one? And if it's also a dependency on osx I suspect fedora needs a depends rule too.